yelyzaveta.kozhevnikova/task2.rb

```diff
@@ -0,0 +1,3 @@
+h1 = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234', yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+h2 = h1.select {|k| k.to_s.downcase.match(/^yes/)}
+puts (h2.size)
```

Comment:
```suggestion
puts h2.size
```

In ruby, we usually don't use parentheses for single argument methods call

http://ruby-for-beginners.rubymonstas.org/bonus/parentheses.html